src/applications/edu-benefits/10203/actions/post-911-gib-status.js

```diff
@@ -1,11 +1,12 @@
+// eslint-disable-next-line @department-of-veterans-affairs/use-workspace-imports
```

Comment:
Altering the import to satisfy the linter caused the cypress tests to break.  Disable linter warning for the import of `apiRequest`